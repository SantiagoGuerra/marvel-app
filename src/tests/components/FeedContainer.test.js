const React = require('react');
const { shallow } = require('enzyme');
const { default: FeedContainer } = require('../../components/FeedContainer');

describe('Test <FeedContainer /> Component', () => {

  const feed = [];

  const filter = {};

  const wrapper = shallow(<FeedContainer feed={feed} filter={filter} />);

  test('should render the component correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
