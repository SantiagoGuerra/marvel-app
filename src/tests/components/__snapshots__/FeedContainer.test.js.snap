// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <FeedContainer /> Component should have data as parameter 1`] = `
<Box
  align="center"
  direction="column"
>
  <Heading
    level={1}
    responsive={true}
    style={
      Object {
        "textTransform": "capitalize",
      }
    }
  />
  <Box
    direction="row"
    gap="small"
    justify="center"
    wrap={true}
  />
</Box>
`;

exports[`Test <FeedContainer /> Component should render the component correctly 1`] = `
<Box
  align="center"
  direction="column"
>
  <Heading
    level={1}
    responsive={true}
    style={
      Object {
        "textTransform": "capitalize",
      }
    }
  />
  <Box
    direction="row"
    gap="small"
    justify="center"
    wrap={true}
  />
</Box>
`;
